templates/16-04/index.html

```diff
@@ -268,4 +310,84 @@ <h5 class="p-code-snippet__title">Server:</h5>
     text-align: left;
   }
 </style>
+
+<script async>
+  function formatDate(date) {
+    let parsedDate = new Date(date);
+    let monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+
+    return parsedDate.getDate() + " " + monthNames[parsedDate.getMonth()] + " " + parsedDate.getFullYear();
+  }
```

Comment:
We have `date-fns` as a dependency which would allow you to simply do
```js
import { format } from "date-fns";
format(d, "dd MMMM YYYY")
```

Reply:
Moved to back-end